updateRepo.sh

```diff
@@ -1,34 +1,53 @@
+#!/bin/bash
+
 echo "Checking if repo needs to be updated"
 export ORG="IBM-Swift"
 export REPO="generator-swiftserver-projects"
 export GH_REPO="github.com/${ORG}/${REPO}.git"
-export BRANCH="init"
+export BRANCHES="init openAPI"
 export projectName="Generator-Swiftserver-Projects"
 
-cd ${TRAVIS_BUILD_DIR}
-mkdir current
-cd current
-git clone -b ${BRANCH} "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${ORG}/${REPO}.git"
-export currentProject=`pwd`
+for BRANCH in $BRANCHES
+do
+  cd "${TRAVIS_BUILD_DIR}" || exit
+  mkdir current
+  cd current || exit
+  git clone -b "${BRANCH}" "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${ORG}/${REPO}.git"
+  currentProject=$(pwd)
+
+  mkdir -p "${TRAVIS_BUILD_DIR}/new/${projectName}"
+  cd "${TRAVIS_BUILD_DIR}/new/${projectName}" || exit
+  if [[ "${BRANCH}" == "init" ]]
+  then
+    yo swiftserver --init --skip-build
+  elif [[ "$BRANCH" == "openAPI" ]]
+  then
+    yo swiftserver --app --spec '{ "appName":"'"$projectName"'", "appType":"scaffold", "appDir":".", "openapi":true, "docker":true, "metrics":true, "healthcheck":true }' --skip-build
+  fi
+
+  rm spec.json
```

Comment:
Do we need to do this in the BRANCH=init case?

Reply:
Good spot, we do not need to do this in that case. I'll fix now.